columnphysics/icepack_shortwave.F90

```diff
@@ -66,10 +66,7 @@ module icepack_shortwave
       implicit none
 
       private
-      public :: run_dEdd, &
-                shortwave_ccsm3, &
-                compute_shortwave_trcr, &
-                icepack_prep_radiation, &
```

Comment:
were the interfaces of these subroutines documented via the autodocumentation script (I did not check) ? if so, ww would want to re-run it to remove them from the public interface doc (and the same for the other subroutines that are made private in other modules).

Reply:
Only the public interfaces in icepack_intfc are auto-documented.  I'll confirm that and also rerun the documentation.